lib/Raven/Configuration.php

```diff
@@ -55,14 +55,16 @@ class Configuration
      * Class constructor.
      *
      * @param array $options The configuration options
+     *
+     * @throws \Symfony\Component\OptionsResolver\Exception\UndefinedOptionsException
+     * @throws \Symfony\Component\OptionsResolver\Exception\AccessException
```

Comment:
As we're in control of the options and how they resolve there should never be a case where the `AccessException` exception is thrown